app/Console/Commands/NotificationsRemoveOld.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+
+class NotificationsRemoveOld extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'notifications:remove:old ${days}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Deletes user notifications over n days old.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $days = $this->argument('days');
+
+        if (is_numeric($days) && $days > 1) {
+            DB::table('notifications')->where('read_at', '<', DB::raw('DATE_SUB(NOW(), INTERVAL '.$days.' day)'))->delete();
```

Comment:
This seems like the best approach and will probably be faster than the alternative you proposed, @marvinroman.